source/gui/addonGui.py

```diff
@@ -129,12 +129,26 @@ def installAddon(self, addonPath, closeAfter=False):
 					prevAddon=addon
 					break
 			if prevAddon:
-				# Translators: A message asking if the user wishes to update a previously installed add-on with this one.
-				if gui.messageBox(_("A version of this add-on is already installed. Would you like to update it?"),
-				# Translators: A title for the dialog  asking if the user wishes to update a previously installed add-on with this one.
-				_("Add-on Installation"),
-				wx.YES|wx.NO|wx.ICON_WARNING)!=wx.YES:
-					return
+				summary=bundle.manifest["summary"]
+				curVersion=prevAddon.manifest["version"]
+				newVersion=bundle.manifest["version"]

```

Comment:
Are we guaranteed that both the old and new addon will have a manifest version parameter? If not what would happen?

Reply:
It wasn't yet possible to install add-ons without a version, and this is not handled gracefully (i.e. an error is raised in the log and there is no visible error). Theoretically, this code indeed would raise an error if the old add-on doesn't have a version in its manifest, but it shouldn't have been possible to install it before.

Having said that, I think it would be good to have a visible error raised if either the old or the new manifest is broken.

Reply:
Is the error for a broken manifest something that you intend to resolve in this PR? I would suggest not.

Reply:
Nope, I"m not intending to do this as it is only partially related to this pr.